wgsl/index.bs

```diff
@@ -3266,6 +3266,7 @@ TODO: deduplicate these tables
   <tr><td>*e* : *T*, *T* is *FloatVec*<td>`isFinite(e)` : bool<*N*>, where *N = Arity(T)*<td>OpIsFinite, or emulate
   <tr><td>*e* : f32<td>`isNormal(e)` : bool<td>OpIsNormal
   <tr><td>*e* : *T*, *T* is *FloatVec*<td>`isNormal(e)` : bool<*N*>, where *N = Arity(T)*<td>OpIsNormal, or emulate
+  <tr><td>*e* : array<*E*><td>`arrayLength(e)` : u32<td>OpArrayLength
```

Comment:
Question: can we make it so this function also works on fixed-size arrays? There is no reason why WGSL implementation couldn't replace it by the length of such array. In this case, it's not going to translate to `OpArrayLength` but instead - just a constant.

Reply:
That seems fine to me.

Reply:
Let's move that to a follow up issue. The group has agreed on arrayLength for Runtime Arrays so let's get that landed and then we can create a PR to discuss extending to  other arrays.